src/Microsoft.VisualStudio.Editors/ResourceEditor/FileWatcher.vb

```diff
@@ -5,7 +5,6 @@ Option Strict On
 Option Compare Binary
 
 Imports Microsoft.VisualStudio.Editors.Common
-Imports Microsoft.VisualStudio.Editors.Common.Utils
 Imports System.IO
```

Comment:
Not going to sort every using without some sort of code fix or rewriter.  This would require me to open every single source file.